src/openvic-simulation/InstanceManager.cpp

```diff
@@ -163,6 +138,11 @@ bool InstanceManager::load_bookmark(Bookmark const* new_bookmark) {
 		map_instance, definition_manager.get_pop_manager().get_pop_types()
 	);
 
+	if(ret) {
```

Comment:
```suggestion
	if (ret) {
```
I'm not sure if we should be `if`-gating the code below, even if there are some issues IMO it's better to try and press ahead. If we don't initialise the map for a new game then we're bascially deciding the game shouldn't proceed, whereas the approach in the rest of the codebase is to press ahead and do as much as we can. If we do decide to abort a game altogether then that decision should be made higher up in the code, from a place where we can half the program altogether and exit in a graceful manner.
If you do keep this `if` statement then it needs a space before the bracket